pkg/standalone/standalone.go

```diff
@@ -1067,6 +1070,20 @@ func getPlacementImageWithTag(name, version string) string {
 	return fmt.Sprintf("%s:%s", name, version)
 }
 
+func checkFallbackImg(imageInfo daprImageInfo, fromDir string) bool {
+	if imageInfo.imageRegistryURL != "" || fromDir != "" {
+		return false
+	}
+	return imageInfo.imageRegistryName == githubContainerRegistryName
+}
+
+func useDefaultReg(privateReg, fromDir string) bool {
+	if privateReg != "" || fromDir != "" {
```

Comment:
What happens if `privateReg` or `fromDir` is a whitespace?
Suggestion : 

`if len(strings.TrimSpace(privateReg)) != 0   ||  len(strings.TrimSpace(fromDir) != 0`